### PR TITLE
Update remove-api-key.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Added complete example

### CODE REVIEW

File: docs/sdks/remove-api-key.md [hunk 1]
```diff
@@ -12,13 +12,50 @@ import TabItem from '@theme/TabItem';
 <TabItem value="js" label="Javascript">
 
 ```js
-// Remove an api key
-try{
-    await client.removeApiKey(keyid);
+const jsc8 = require("jsc8");
+
+// Email and Password to authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/sdks/remove-api-key.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/remove-api-key.md [hunk 2]
```diff
@@ -12,13 +12,50 @@ import TabItem from '@theme/TabItem';
...
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+
+// Or use one of the following authentication methods and remove the commenting.
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/sdks/remove-api-key.md]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!

**Review by diana-macrometa — APPROVED** [review]
LGTM

**Comment by diana-macrometa** [comment]
@edgargarciamacrometa, is this ready to merge?

**Comment by edgargarciamacrometa** [comment]
@diana-macrometa Yes

**Pull request merged**